### ISSUE TITLE
Avast One Alerts

### ISSUE THREAD

**Issue by RickSeiden** [qmk/qmk_toolbox#453]
Hello,

I downloaded v0.3.0 qmk_toolbox.exe from right here on Github and when I ran it, Avast One went crazy telling me all kinds of files are infected with Win64:EVO-gen [Trj].  I **know** that this is not intentional, and I **think** this is probably a bunch of false positives, but I thought I should report it anyway.  I'm going to try downloading the installer.

![image](https://github.com/qmk/qmk_toolbox/assets/2730583/1774b6ac-8a9b-4e86-97ca-766cf4544f3a)

**Comment by fauxpark** [comment]
Yes, it is a false positive. The flashing tools bundled with the Windows Toolbox are taken directly from the MSYS2 MINGW64 packages, unmodified (you can check the hashes of the .exe if you want): https://packages.msys2.org/package/mingw-w64-x86_64-dfu-programmer?repo=mingw64

So unless someone has managed to sneak some malware into dfu-programmer itself, or replace the binary in the MSYS2 repo, it should be safe.

**Comment by RickSeiden** [comment]
I ran the installer instead and had no issues with that.  I know that QMK is a very well respected project, so I don't believe for even a nanosecond that anyone here would intentionally infect their code with a virus.  Avast One, however, not so much.  Because it blocked programs that had a false positive, I couldn't use it if I wanted to.
